src/kubeobjects/labels.go

```diff
@@ -1,5 +1,22 @@
 package kubeobjects
 
+import "github.com/Dynatrace/dynatrace-operator/src/version"
+
+const (
+	AppNameLabel      = "app.kubernetes.io/name"
+	AppCreatedByLabel = "app.kubernetes.io/created-by"
+	AppComponentLabel = "app.kubernetes.io/component"
+	AppVersionLabel   = "app.kubernetes.io/version"
+	FeatureLabel      = "operator.dynatrace.com/feature"
```

Comment:
I don't think its necessary, we already have `app.kubernetes.io/name: dynatrace-operator` label on everything that has to do with the operator, and that already distinguishes it from the old one. 

Also *dynatrace* would be present twice in the name which feels a bit redundant.

Maybe we could rename it to `component.dynatrace.com/feature` as we have the `app.kubernetes.io/component` label as well. 
So for the oneagent it would look something like
```yaml
    app.kubernetes.io/name: dynatrace-operator
    app.kubernetes.io/version: "0.5.0"
    app.kubernetes.io/component: oneagent
    component.dynatrace.com/feature: cloud_native_fullstack  #(this is snake_case because the deployment metadata is also snake_case or something similar :) )
```